packages/forms/src/components/FormActions/index.tsx

```diff
@@ -9,15 +9,17 @@ export type Props = Omit<BaseProps, 'disabled' | 'processing'>;
 /** `FormActions` automatically connected to the parent `Form`. */
 export default function FormActions(props: Props) {
   const context = useContext(FormContext);
+  let baseProps: BaseProps = props;
```

Comment:
If you spread here then the assign code doesn't change much.

Reply:
will update